[CS2103T-T09-3] ClerkPro

SakuraBlossom opened the pull request:
@SakuraBlossom
@woon17
@wongsm7
@CarbonGrid

blimyj commented on docs/DeveloperGuide.adoc:
```diff
@@ -297,58 +298,162 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* *` |clerk|quick-fill the command box with history commands|inputting last few commands is easier
+
+|`* * *` |clerk|find out the upcoming appointments for given patients. |
```
Is there supposed to be a full stop here?

blimyj commented on docs/DeveloperGuide.adoc:
```diff
@@ -297,58 +298,162 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+|`* *` |experienced user|create custom aliases to commands |
+
+|`* *` |receptionist|use the undo and redo feature to remedy any mistakes.|
```
Do you want to consider splitting the function into function and benefit?
|undo and redo actions | to remedy any mistakes|

SendorasRight replied:
it would be good if you define the difference/relationship between 'receptionist' and 'clerk'. if 'receptionist' is a more specialised version of a 'clerk', an actor generalisation could be defined

blimyj commented on docs/DeveloperGuide.adoc:
```diff
@@ -297,58 +298,162 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+|`* * *` |clerk|take note of the doctors that are on-shift | effectively direct patients to available doctors.
+
+|`* *` |experienced user|create custom aliases to commands |
```
Do you want to add in the benefit of this function?
eg. |to save time|

blimyj commented on docs/DeveloperGuide.adoc:
```diff
@@ -297,58 +298,162 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+|`* * *` |clerk|use the appointment scheduler | schedule appointments for my patients
+
+|`* * *` |clerk|save time managing the queue | have more time to do my own work
```
Do you think you can be more specific on how this application can help the user save time while managing the queue?

blimyj commented on docs/DeveloperGuide.adoc:
```diff
@@ -297,58 +298,162 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+.  Should be able to hold up to 10000 persons and 10000 events without a noticeable sluggishness in performance for typical usage.
 .  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+.  No internet required.
+.  System allows user to undo incorrect or accidental destructive actions
```
Do you think this should be a functional requirement instead?

SendorasLeft commented on docs/DeveloperGuide.adoc:
```diff
@@ -297,50 +298,147 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+*Extensions*
+
+* 2a. Actor inputs invalid format
```
It might be good if you use consistent terms throughout to refer the the 'Actor' or 'User'. It's a little confusing to interchange them in a single UC

chensu2436 commented on docs/DeveloperGuide.adoc:
```diff
@@ -297,58 +298,162 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
++
+Use case ends.
+
+* 2a. ClerkPro requests for correct input format. +
```
Should it be *2a1?

chensu2436 commented on docs/DeveloperGuide.adoc:
```diff
@@ -297,58 +298,162 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+*MSS*
+
+1.  Patient calls to change appointment date
+2.  User checks whether the current appointment can be postponed.
```
Should here be a response shown by ClerkPro?

chensu2436 commented on docs/DeveloperGuide.adoc:
```diff
@@ -297,58 +298,162 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+*MSS*
+
+1.  Patient wants to leave
```
Should it be a pre-condition instead since there is no interaction between the patient and the system?

cheongliyuan commented on docs/DeveloperGuide.adoc:
```diff
@@ -297,58 +298,162 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+*Extensions*
+
+* 2a. Current appointment date is invalid +
++
```
Should ClerkPro give an option to try again for the input, instead of ending case here?

chensu2436 commented on docs/DeveloperGuide.adoc:
```diff
@@ -297,58 +298,162 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+*MSS*
+
+1.  Patient wants to set next appointment
+2.  User requests to set the patient's appointment date
```
Should there be an extension for appointment date not available?

SendorasRight commented on docs/DeveloperGuide.adoc:
```diff
@@ -297,58 +298,162 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+*Extensions*
+
+* 2a. Actor inputs invalid format
```
should use consistent terms to refer to the same role. User and Actor both seem to refer to the same person

Luk723 commented on docs/DeveloperGuide.adoc:
```diff
@@ -297,58 +298,162 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+|`* *` |clerk|tag patient with known allergies. | keep track of their allegies
+
+|`* *` |clerk|cancel appointments for patients | free up appointment time slots
```
should organise user stories by 'must haves' and 'nice to haves'

blimyj reviewed (commented):
Good job! All three parts are present for most of the user stories. 😄

sianghwee commented on docs/DeveloperGuide.adoc:
```diff
@@ -117,9 +118,15 @@ image::EnqueueSequenceDiagram.png[]
+.Interactions Inside the Logic Component for the `cancelappt 1` Command
+image::CancelAppointmentSequenceDiagram.png[]
+
+NOTE: The lifeline for `CancelAppCommandParser` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
```
Perhaps you can consider using Photoshop to remove the excess dotted line instead of putting a comment about it.

sianghwee commented on docs/DeveloperGuide.adoc:
```diff
@@ -117,9 +118,15 @@ image::EnqueueSequenceDiagram.png[]
 .Interactions inside the Logic Component for the `next 1` Command
```
Do you think that it might be better to have a gap between the logic and model class to make it more readable? You can also consider splitting up the diagram as the fonts are too small at the moment.

sianghwee reviewed (commented):
Hi, just a little comments on the UML diagrams.

blimyj commented on docs/DeveloperGuide.adoc:
```diff
@@ -111,6 +111,19 @@ image::DeleteSequenceDiagram.png[]
 
 NOTE: The lifeline for `DeleteCommandParser` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
```
Do you want to color out the lifeline for the DeleteCommandParser below the destroy marker so that the diagram is accurate?

blimyj commented on docs/DeveloperGuide.adoc:
```diff
@@ -111,6 +111,19 @@ image::DeleteSequenceDiagram.png[]
 NOTE: The lifeline for `DeleteCommandParser` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
 
+.Interactions inside the Logic Component for the `enqueue 001A` Command
+image::EnqueueSequenceDiagram.png[]
```
Do you want to split the diagram horizontally so that it is easier to read the labels on the diagram?
I feel that the diagram is currently a little hard to read.

blimyj commented on docs/DeveloperGuide.adoc:
```diff
@@ -111,6 +111,19 @@ image::DeleteSequenceDiagram.png[]
+image::EnqueueSequenceDiagram.png[]
+
+.Interactions inside the Logic Component for the `next 1` Command
+image::NextSequenceDiagram.png[]
```
Do you want to color out the lifeline for the NextCommandParser below the destroy marker so that the diagram is accurate or add a note to acknowledge the limitation of PlantUML like you did for the other diagrams?

blimyj commented on docs/DeveloperGuide.adoc:
```diff
@@ -111,6 +111,19 @@ image::DeleteSequenceDiagram.png[]
+Given below is the Sequence Diagram for interactions within the `Logic` component for the `execute("cancelappt 1")` API call.
+
+.Interactions Inside the Logic Component for the `cancelappt 1` Command
+image::CancelAppointmentSequenceDiagram.png[]
```
Do you want to color out the lifeline for the CancelAppCommandParser below the destroy marker so that the diagram is accurate?

blimyj reviewed (commented):
The developer guide is quite clear overall! Great work!

SakuraBlossom committed:
Clarifies edit person error message

SakuraBlossom committed:
Fixes listing behaviour

SakuraBlossom committed:
Fixes test cases

SakuraBlossom committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-parser-fixes

SakuraBlossom committed:
Adds space after commands selected through AutoComplete

SakuraBlossom committed:
Refactor Logic Manager

woon17 committed:
edits timing checking in ParserUtil

woon17 committed:
fixes test case

woon17 committed:
Merge pull request #211 from woon17/master

Fixes timing parser and edits result message

SakuraBlossom committed:
Merge pull request #210 from SakuraBlossom/branch-parser-fixes

Fixes Parser Behaviour

woon17 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master'

wongsm7 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-test-cases

wongsm7 committed:
Updates author of files, edits test cases

wongsm7 committed:
Updates author for test files

SakuraBlossom committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master'

wongsm7 committed:
Changes name

wongsm7 committed:
Updates layout image

CarbonGrid committed:
Updates Documentation & UI Tests.

CarbonGrid committed:
Merge https://github.com/AY1920S1-CS2103T-T09-3/main

CarbonGrid committed:
Removes dead code in UI & Adds UI Test.

CarbonGrid committed:
Merge pull request #217 from CarbonGrid/master

Updates Documentation, Tests & Logging.

SakuraBlossom committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-parser-fixes

woon17 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master'

SakuraBlossom committed:
Fixes behaviour of eager evaluation

wongsm7 committed:
Updates sample person data

SakuraBlossom committed:
Adds author tags

wongsm7 committed:
Merge pull request #216 from wongsm7/branch-test-cases

Updates test cases, edits layout image, adds author to files

wongsm7 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-sampledata

SakuraBlossom committed:
Merge pull request #218 from SakuraBlossom/branch-parser-fixes

Fixes behaviour of eager evaluation

wongsm7 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-sampledata

wongsm7 committed:
Fixes Sample data generator

wongsm7 committed:
Updates UGDG about command usages

CarbonGrid committed:
Fixes AutoCompleteOverlay persisting after CommandBox loses focus.
StatusBarFooter is now meaningful.
Updated DeveloperGuide.adoc.
Updated carbongrid PPP.

wongsm7 committed:
Fixes Check Style

wongsm7 committed:
Adds author tag to SamplePersonDataUtil class

wongsm7 committed:
Merge pull request #219 from wongsm7/branc-sampledata

Updates sample person data

woon17 committed:
edits result message and updates UG

changes "recursive" to "reoccurring"

woon17 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master'

CarbonGrid committed:
Merge https://github.com/AY1920S1-CS2103T-T09-3/main

SakuraBlossom committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-docs

woon17 committed:
edits DG

woon17 committed:
edits duty shift in DG

woon17 committed:
edits appointments command

CarbonGrid committed:
Updates Ui.png and Layout.png.

woon17 committed:
edits test case message

CarbonGrid committed:
Fixes blurry Layout.png.

woon17 committed:
edits index to be 0

CarbonGrid committed:
Merge pull request #220 from CarbonGrid/master

Documentation update and Feature Improvement.

woon17 committed:
Merge pull request #221 from woon17/master

Adds duty shift in UG and edits result message

SakuraBlossom committed:
Updates UG about Reactive search and appointment's timings

CarbonGrid committed:
Makes darkness more prominent.

SakuraBlossom committed:
Renames AddressBookParser to SysremCommandParser

woon17 committed:
updates message in ParserUtil

SakuraBlossom committed:
Updates Intro Paragraph of UG

SakuraBlossom committed:
Updates the design-logic section of DG

wongsm7 committed:
Edits DeveloperGuide

CarbonGrid committed:
Double checked and Improved line 267 to 360 of DG.

CarbonGrid committed:
Corrected wrong command word.

CarbonGrid committed:
Merge https://github.com/AY1920S1-CS2103T-T09-3/main

wongsm7 committed:
Edits developer guide

CarbonGrid committed:
Revolves unresolved merge conflicts.

CarbonGrid committed:
Modified as per request.

wongsm7 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-userguide

wongsm7 committed:
Edits userguide

wongsm7 committed:
Merge pull request #226 from wongsm7/branch-userguide

Edits userguide

CarbonGrid committed:
Merge https://github.com/AY1920S1-CS2103T-T09-3/main

CarbonGrid committed:
More modifications for precise terminology.

CarbonGrid committed:
Merge pull request #225 from CarbonGrid/master

Audited and Improved DG.

wongsm7 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-DG

woon17 committed:
edits ackappt command result message

wongsm7 committed:
Edits developer guide

wongsm7 committed:
Edits developer guide #2

SakuraBlossom committed:
Updates Logic component in DG

wongsm7 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-DG

wongsm7 committed:
Edits developer guide #3

wongsm7 committed:
Merge pull request #224 from wongsm7/branch-DG

Edits DeveloperGuide

SakuraBlossom committed:
Updates Model section DG

woon17 committed:
edits ackappt command to make sure only acknowledge today's appointment

woon17 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master'

SakuraBlossom committed:
Updates description of UID in DG

SakuraBlossom committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-docs

woon17 committed:
updates DG

woon17 committed:
edits result message for appointments command

woon17 committed:
edits test cases

SakuraBlossom committed:
Remove history limit

SakuraBlossom committed:
Drafts undo/redo feature in DG

CarbonGrid committed:
Adds more darkness with a hint of colours.

CarbonGrid committed:
Improves AutoCompleteOverlay alignment.

CarbonGrid committed:
Adds more elegant styling.

CarbonGrid committed:
Merge https://github.com/AY1920S1-CS2103T-T09-3/main into branch-UiRework

wongsm7 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-DG

wongsm7 committed:
Updates model class diagram

wongsm7 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-ppp

woon17 committed:
edits activity diagram if statement

woon17 committed:
edits appointment and shift's result message

woon17 committed:
edits ackappt command in UG

woon17 committed:
Merge pull request #228 from woon17/master

Updates DG and edits diagram

woon17 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master'

woon17 committed:
changes "missing" to "missed"

wongsm7 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-ppp

wongsm7 committed:
Edits ppp

wongsm7 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-userguide

CarbonGrid committed:
Improves threading order.
Sets new Ui.png and Layout.png.

wongsm7 committed:
Edits user guide

wongsm7 committed:
Edits user guide #2

wongsm7 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-sampledata

CarbonGrid committed:
Merge pull request #222 from CarbonGrid/branch-UiRework

Ui Recolor.

wongsm7 committed:
Adds citation for sample data

wongsm7 committed:
Edits sample data

wongsm7 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-sampledata

wongsm7 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-DG

wongsm7 committed:
Edits sequence diagrams for enqueue and next commands

woon17 committed:
creates timing used for example

woon17 committed:
edits checking for editshift command

woon17 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master'

SakuraBlossom committed:
Updates undo/redo guide

wongsm7 committed:
Merge pull request #230 from wongsm7/branc-sampledata

Adds citation for sample data

wongsm7 committed:
Merge pull request #229 from wongsm7/branch-userguide

Edits user guide

wongsm7 committed:
Merge pull request #231 from wongsm7/branch-DG

Edits UML diagrams for enqueue and next commands

wongsm7 committed:
Merge pull request #232 from wongsm7/branch-ppp

Edits ppp

SakuraBlossom committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-docs

woon17 committed:
Merge pull request #233 from woon17/master

Updates example timing and timing checking for editshift command

woon17 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master'

SakuraBlossom committed:
Fixes check style for puml file

SakuraBlossom committed:
Merge pull request #234 from SakuraBlossom/branch-docs

Edits DG Logic, model and Storage

woon17 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master'

SakuraBlossom committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-docs

SakuraBlossom committed:
Updates DG description of Undo/Redo feature

SakuraBlossom committed:
Updates UndoRedo diagrams

SakuraBlossom committed:
Updates Architecture Diagram

CarbonGrid committed:
Updates Documentation and Improve threading in LogicManager.

woon17 committed:
edits UG and DG

CarbonGrid committed:
Adds sleep back.

woon17 committed:
removes unused import

CarbonGrid committed:
Adds missing newline.

CarbonGrid committed:
Removes unused imports.

CarbonGrid committed:
Reduces sample data back to 1000 patients.

woon17 committed:
Edits MainAppTest

woon17 committed:
Edits code

SakuraBlossom committed:
Update DG

SakuraBlossom committed:
Fixes behavour of Eager evaluation to conform to DG specifications

woon17 committed:
Edits code

woon17 committed:
Merge pull request #237 from woon17/master

Updates DG and UG

CarbonGrid committed:
Merge pull request #236 from CarbonGrid/master

Updates UI & Documentation and Refactors threading in LogicManager.

SakuraBlossom committed:
Merge pull request #235 from SakuraBlossom/branch-docs

Updates DG about implemented features Undo/Redo

CarbonGrid committed:
Update PPP and tags in UG.

SakuraBlossom committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master'

SakuraBlossom committed:
Tidy up unit test cases for list patient command

SakuraBlossom committed:
Update authorship tags

SakuraBlossom committed:
Adds ui test case for edit staff command

CarbonGrid committed:
Improves Documentation & error handling in MainWindow sizing.

SakuraBlossom committed:
Merge pull request #240 from SakuraBlossom/branch-unit-test

Updates unit test cases

wongsm7 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-bugfix

SakuraBlossom committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-docs

woon17 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master'

wongsm7 committed:
Fixes bugs for queue commands, edit commands

woon17 committed:
edits for code quality

SakuraBlossom committed:
Update code contribution link

wongsm7 committed:
Updates test cases and fix stylechecks

wongsm7 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-ppp

wongsm7 committed:
Updates ppp

wongsm7 committed:
Edits message for edit commands

wongsm7 committed:
Edits typo error

wongsm7 committed:
Adds examples for UG

wongsm7 committed:
Edits message for edit commands #2

wongsm7 committed:
Adds newline at EOF

wongsm7 committed:
Merge pull request #241 from wongsm7/branch-bugfix

Fixes bug for queue and edit commands

wongsm7 committed:
Merge pull request #242 from wongsm7/branch-ppp

Edits ppp, add tags for UG and DG, examples for UG

woon17 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master'

wongsm7 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-bugfix

woon17 committed:
add tags in UG,DG

woon17 committed:
edits PPP

woon17 committed:
Merge pull request #243 from woon17/Branch-PPP

Updates PPP and adds tags in UG and DG

woon17 committed:
updates "Name"

woon17 committed:
updates PPP and UG

woon17 committed:
deletes all todo tasks

woon17 committed:
updates URL

woon17 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into Branch-PPP

wongsm7 committed:
Updates focus for enqueue command

wongsm7 committed:
Fixes javadoc

woon17 committed:
refactor methods and updates their javaDoc

woon17 committed:
adds @author

woon17 committed:
refactor appointments' file name

woon17 committed:
edits error message for shift command

fails in checking idx

woon17 committed:
deletes all example for appointments and shifts

if give example in UG, timing is fix and maybe fail in future

woon17 committed:
Merge pull request #246 from woon17/Branch-code-quality

Code quality checking

wongsm7 committed:
Edits UG for command format

CarbonGrid committed:
Improves Documentation & error handling of window sizing in MainWindow.

woon17 committed:
refactor back

woon17 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into Branch-PPP

CarbonGrid committed:
Remove contribution in UG. Pictures too large.

CarbonGrid committed:
Fixes Use Case mis-indexing.

CarbonGrid committed:
Merge pull request #239 from CarbonGrid/master

Updates PPP and tags in UG.

wongsm7 committed:
Updates DG

woon17 committed:
throw exception when the editEvent is the same timing

woon17 committed:
edits result messages

wongsm7 committed:
Edits ppp

woon17 committed:
updates appointment's aspect

SakuraBlossom committed:
Updates DG about reactive search implementation

wongsm7 committed:
Updates css for printing

CarbonGrid committed:
Re-adds contribution and fixes link in pdf view.

SakuraBlossom committed:
Update reactive search seq diagrams

CarbonGrid committed:
Fixes url link to release webpage.

wongsm7 committed:
Updates ppp and DG

wongsm7 committed:
Removed pdf file

SakuraBlossom committed:
Update SakuraBlossom's PPP UG portion

wongsm7 committed:
Merge pull request #245 from wongsm7/branch-bugfix

Updates focus for enqueue command, fixes code

SakuraBlossom committed:
Update PPP

wongsm7 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-bugfix

CarbonGrid committed:
Adds Class Diagram for AutoCompleter.

SakuraBlossom committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-docs

wongsm7 committed:
Updates developer guide

CarbonGrid committed:
Merge pull request #248 from CarbonGrid/master

Re-adds contribution and fixes link in pdf view.

SakuraBlossom committed:
Merge pull request #247 from SakuraBlossom/branch-docs

Updates DG about Reactive Search

woon17 committed:
updates PPP

wongsm7 committed:
Edits ppp

wongsm7 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-bugfix

wongsm7 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-ppp

wongsm7 committed:
Edits ppp

SakuraBlossom committed:
updates to PPP

woon17 committed:
fixs bug for editappt

woon17 committed:
enlarges command result box

wongsm7 committed:
Merge pull request #250 from wongsm7/branch-ppp

Edits ppp

CarbonGrid committed:
Quick fix UG link.

woon17 committed:
Merge pull request #244 from woon17/Branch-PPP

Updates PPP, DG and UG

wongsm7 committed:
Merge pull request #249 from wongsm7/branch-bugfix

Edits developer guide

wongsm7 committed:
Edits ppp

wongsm7 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-ppp

CarbonGrid committed:
Adds asciidoc hints.

woon17 committed:
deletes pdf file

woon17 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into Branch-PPP

CarbonGrid committed:
Merge pull request #251 from CarbonGrid/master

Quick fix UG link.

wongsm7 committed:
Merge pull request #253 from woon17/Branch-PPP

deletes pdf file

wongsm7 committed:
Merge pull request #252 from wongsm7/branch-ppp

Edits ppp

woon17 committed:
edits PPP

woon17 committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into Branch-PPP

CarbonGrid committed:
Corrected tagging in docs/DeveloperGuide.

CarbonGrid committed:
Adds more asciidoc hints.

CarbonGrid committed:
Amend hints.

woon17 committed:
adds "command"

SakuraBlossom committed:
Updates DG

SakuraBlossom committed:
Delete PDF files

woon17 committed:
Merge pull request #254 from woon17/Branch-PPP

Updates PPP

CarbonGrid committed:
Merge pull request #255 from CarbonGrid/master

Adds Asciidoc hints.

SakuraBlossom committed:
Fixes typo in UG

SakuraBlossom committed:
Update individual PPP

SakuraBlossom committed:
Near last push

SakuraBlossom committed:
Merge remote-tracking branch 'AY1920S1-CS2103T-T09-3/master' into branch-docs

SakuraBlossom committed:
Fixes check style

SakuraBlossom committed:
Merge pull request #256 from SakuraBlossom/branch-docs

Updates PPP for SakuraBlossom
